django/santropolFeast/member/models.py

```diff
@@ -489,6 +489,13 @@ def notes(self):
         return self.client_notes.all()
 
     @property
+    def scheduled_statuses(self):
+        """
+        Returns scheduled status associated to this client
+        """
+        return self.client_scheduled_statuses.all()
+
```

Comment:
@philippemouchel if you change the declaration in the model `ClientScheduledStatus` from 
```
client = models.ForeignKey(
        Client,
        on_delete=models.CASCADE,
        related_name='client_scheduled_statuses'
    )
```
to 
```
client = models.ForeignKey(
        Client,
        on_delete=models.CASCADE,
        related_name='scheduled_statuses'
    )
```

you don't really need to define this property, it will remain the same in the others parts of the code, you could access to the client statuses using `self.scheduled_statuses.all()` in the class or `client.scheduled_statuses.all()` in the object. You also will be able to filter the status, I think it will remain more flexible.